### PR TITLE
Nav bar test

### PR DESCRIPTION
**Pull request by mattfreeman-london**
All tests are currently passing so submitting branch for review.

### CODE REVIEW

File: src/components/navBarItem/navBarItem.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import "bootstrap/dist/css/bootstrap.css"
 
-export const NavBarItem = (props) => {
+export const NavBarItem = (props: any) => {
```
Comment (Gina-Frankel) [inline src/components/navBarItem/navBarItem.tsx:4]: Can we replace any with something more specific?

File: src/components/navBar/navBar.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from "react"
+import { shallow } from "enzyme"
+import { NavBar } from "../index"
+
+it("renders the NavBar correctly", () => {
+    let wrapper = shallow(<NavBar ></NavBar>)
+    expect(wrapper).toHaveLength(1)
+})
```
Comment (Gina-Frankel) [inline src/components/navBar/navBar.spec.tsx:8]: Add a new line at end of file

File: src/components/bookFrontCover/bookFrontCover.spec.tsx [hunk 3]
```diff
@@ -3,6 +3,6 @@ import { shallow } from "enzyme"
 import { BookFrontCover } from "../index"
 
 it("renders a page correctly", () => {
-  let wrapper = shallow(<BookFrontCover />)
+  let wrapper = shallow(<BookFrontCover greeting="hello" jobTitle="developer" careerAim="data ops engineer" />)
   expect(wrapper).toHaveLength(1)
 })
```
Comment (Gina-Frankel) [inline src/components/bookFrontCover/bookFrontCover.spec.tsx:8]: 👍

File: src/components/layout/index.spec.tsx [hunk 4]
```diff
@@ -4,8 +4,7 @@ import { Layout } from "./index"
 
 it("renders a page correctly", () => {
   let wrapper = shallow(
-    <Layout title="hi">
-      <div></div>
+    <Layout >
```
Comment (Gina-Frankel) [inline src/components/layout/index.spec.tsx:7]: 👍

File: src/content/chapters.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+#     -"The Start"
+#     -"Working Life"
+#     -"Eureka"
+#     -"Learning"
+#     -"Projects"
+#     -"Future"
+#     -"Contact"
+
+test: "test string"
+---
```
Comment (Gina-Frankel) [inline src/content/chapters.md:14]: What is this file for?

Comment (benfurber) [reply src/content/chapters.md:14]: Same question.

**Review by benfurber — CHANGES_REQUESTED** [review]
Why was just a big yarn.lock change required?

File: src/content/chapters.md (hunk 5 above)
Comment (mattfreeman-london) [reply src/content/chapters.md:14]: I have no idea - wasn't aware that it had changed at all. Can take it out again.

File: src/components/navBarItem/navBarItem.tsx (hunk 1 above)
Comment (mattfreeman-london) [reply src/components/navBarItem/navBarItem.tsx:4]: Sure - can you suggest what? I don't understand props well enough to know what should be swapped in.

**Pull request closed without merging**